canopeum_backend/canopeum_backend/views.py

```diff
@@ -309,15 +335,19 @@ def patch(self, request, siteId):
                 visible_map=json.loads(request.data["visibleMap"]),
             )
 
-            for tree_type_json in request.data.getlist("species"):
+            # TODO: Are we sure about getlist ?
+            # If this is correct, consider raising an issue upstream
+            for tree_type_json in request.data.getlist("species"):  # type:ignore[attr-defined] # pyright: ignore[reportAttributeAccessIssue]
```

Comment:
@BesOlivierBouchard Since you added this. Can you validate whether this is valid code?

Reply:
From what I saw online, this is the way to retrieve a "list" from "data". Doing it with the get() or the data[] will not work properly.